docs/examples/plotting/auto_color_scale.ipynb

```diff
@@ -42,13 +42,12 @@
    "metadata": {},
    "source": [
     "## Introduction to the problem and risks\n",
-    "It is a fairly common annoyance that a heat map plot of ones dearest data does not show any of its details due to some outliers that push the range of the color scale beyond the desired limits, as it is shown in the image below:\n",
+    "It is a fairly common issue that a heat map plot of a given data does not properly show the details due to some outliers that push the range of the color scale beyond the desired limits, as it is shown in the image below:\n",
```

Comment:
another 'the' ;-)
The word 'data' needs a definite article. (https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwjb6pbu49LlAhVRJFAKHQeECCUQFjABegQICxAF&url=https%3A%2F%2Fell.stackexchange.com%2Fquestions%2F115842%2Fdefinite-article-before-countable-and-uncountable-nouns&usg=AOvVaw2GpyqeQio16xf4rTuqhFQj)
Maybe lets just leave 'data' out. (That is the only thing a heat map plot could possibly show.)
- of a given data
'some details'